client/src/components/KeyConceptList/KeyConceptList.stories.js

```diff
@@ -0,0 +1,31 @@
+import _ from "lodash";
+import React from "react";
+
+import { TM } from "src/components/index";
+
+import { KeyConceptList } from "./KeyConceptList";
+
+export default {
+  title: "KeyConceptList",
+  component: KeyConceptList,
+};
+
+const Template = (args) => <KeyConceptList {...args} />;
+
+const question_answer_pairs = _.map(
+  [
+    "MtoM_tag_warning_reporting_level",
+    "MtoM_tag_warning_resource_splitting",
+    "MtoM_tag_warning_double_counting",
+  ],
+  (key) => [
+    <TM key={key + "_q"} k={key + "_q"} />,
+    <TM key={key + "_a"} k={key + "_a"} />,
+  ]
+);
```

Comment:
I'd suggest passing in hard coded strings, not using the text maker here. Lorem ipsum, or something. That should be the rule for storybooks in general, unless the component specifically needs needs text maker to work. Using text keys here means this story is coupled to this text content and could be broken by changes to those keys.

Reply:
Done!